code/glTF/glTFExporter.cpp

```diff
@@ -173,6 +174,62 @@ static void IdentityMatrix4(glTF::mat4& o)
     o[12] = 0; o[13] = 0; o[14] = 0; o[15] = 1;
 }
 
+template<typename T>
+void SetAccessorRange(Ref<Accessor> acc, void* data, unsigned int count,
+	unsigned int numCompsIn, unsigned int numCompsOut)
+{
+	ai_assert(numCompsOut <= numCompsIn);
+
+	// Allocate and initialize with large values.
+	for (unsigned int i = 0 ; i < numCompsOut ; i++) {
+		acc->min.push_back( std::numeric_limits<double>::max());
+		acc->max.push_back(-std::numeric_limits<double>::max());
```

Comment:
There is also `std::numeric_limits<double>::min()`, could be better than `-`

Reply:
`float` and `double` `min` is different from integer mins. It's actually the "smallest" accurately represented non-zero number and its not negative. It is super confusion and a bad design choice in C/C++ stdlib, but that's the way it is.

Reply:
you're right , it's `lowest`

Reply:
`lowest` would work, but it was added in C++11. I'm not sure what version of C++ this project is targeting actually.